tritium-metrics/src/main/java/com/palantir/tritium/metrics/TaggedMetricsScheduledExecutorService.java

```diff
@@ -248,7 +248,7 @@ public T call() throws Exception {
     private static MetricName createMetricName(String metricName, String name) {
         return MetricName.builder()
                 .safeName(MetricRegistry.name("executor", metricName))
-                .putSafeTags("name", name)
+                .putSafeTags("executor-name", name)
```

Comment:
How about "executor"?

I think I used `"name"` for the SSLContext metrics as well, would you mind updating that to `context` in this PR as well?